nautobot/core/jobs/bulk_actions.py

```diff
@@ -0,0 +1,156 @@
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import (
+    FieldDoesNotExist,
+    ObjectDoesNotExist,
+    PermissionDenied,
+    ValidationError,
+)
+from django.db.models import ManyToManyField
+
+from nautobot.core.forms.utils import restrict_form_fields
+from nautobot.core.utils.lookup import get_filterset_for_model, get_form_for_model
+from nautobot.extras.context_managers import deferred_change_logging_for_bulk_operation
+from nautobot.extras.jobs import (
+    BooleanVar,
+    Job,
+    JSONVar,
+    ObjectVar,
+    RunJobTaskFailed,
+)
+from nautobot.extras.utils import remove_prefix_from_cf_key
+
+
+class BulkEditObjects(Job):
+    """System Job to bulk Edit objects."""
+
+    content_type = ObjectVar(
+        model=ContentType,
+        description="Type of objects to update",
+    )
+    post_data = JSONVar(description="Data to update with i.e request.POST")
```

Comment:
I kind of prefer the clarity that the delete job has with having a distinct `pk_list` parameter. Would that be difficult to implement in this case?

Reply:
I could change it to `form_data`, as `pk_list` is part of the `BulkEdit` form fields.

Reply:
Understood - I was wondering if it'd make sense to split it out from the rest of the form data into a separate Job input. Fine if not.